engine/settings/base.py

```diff
@@ -220,9 +220,7 @@ class DatabaseTypes:
 CACHES = {
     "default": {
         "BACKEND": "redis_cache.RedisCache",
-        "LOCATION": [
-            REDIS_URI,
-        ],
+        "LOCATION": REDIS_URI,
```

Comment:
this is a safe change as you can also simply provide a string to `LOCATION`. [From the docs](https://docs.djangoproject.com/en/4.2/topics/cache/#:~:text=In%20order%20to%20supply%20a%20username%20and%20password%2C%20add%20them%20in%20the%20LOCATION%20along%20with%20the%20URL):

> ```python3
> CACHES = {
>     "default": {
>         "BACKEND": "django.core.cache.backends.redis.RedisCache",
>         "LOCATION": "redis://username:password@127.0.0.1:6379",
>     }
> }
> ```